bin/update_config.sh

```diff
@@ -14,7 +14,7 @@ cd config_checkout
 
```

Comment:
Change make sense, but wondering if it's a redundant file.

Is this file a remnant of the Deis days, I wonder...? @pmac will surely know

Reply:
A quick grep and I don't see it referenced anywhere, in bedrock or webservices-infra.

Reply:
I don't think it was Deis, but it was for our k8s cluster before SRE took over hosting on their cluster. I'm confident we can delete this file.

Reply:
Thanks y'all!